tests/mocks/limsmock.py

```diff
@@ -35,10 +49,17 @@ class LimsSample(BaseModel):
     received: str = None
     source: str = None
     priority: str = None
+    udfs: LimsUDF = LimsUDF()
+
+
+class LimsArtifactObject(BaseModel):
+    parent_process: LimsProcess = LimsProcess.COVID_POOLING_STEP
+    type: LimsArtifactTypes = LimsArtifactTypes.ANALYTE
+    samples: list[LimsSample] = []
 
 
 class MockLimsAPI(LimsAPI):
```

Comment:
Also just for the record: we should avoid using the Mock classes in the future and use Mocker instead mocking specific methods of the original class (`mock.patch.object()`)

Reply:
I see, thanks for the info! Do you have tips on where to read more about this, by the way? 🤔

Reply:
Sent by slack, resolving ☺️